core/apps/ame/src/app/components/editor-canvas/sidebar/sidebar.component.ts

```diff
@@ -232,6 +255,28 @@ export class EditorCanvasSidebarComponent implements AfterViewInit, OnInit, OnDe
 
   private loadNamespace(namespaceFolder: string, namespaceFile: string) {
     const cachedFile = this.editorService.loadExternalAspectModel(`${namespaceFolder}:${namespaceFile}`);
+
+    const duplicateElements = [];
+
+    this.namespaceCacheService
+      .getCurrentCachedFile()
+      .getAllElements<BaseMetaModelElement>()
+      .forEach(currentFileElement =>
+        cachedFile.getAllElements<BaseMetaModelElement>().forEach(cachedFileElement => {
```

Comment:
`cachedFileElement` is shadowing the variable from first forEach

Reply:
The first one is named "currentFileElement" and the second "cachedFileElement". 
Or what you mean?

Reply:
I am sorry. I thought they had the same name.
This comment is useless then. I will close it